app/src/main/java/com/codesoom/assignment/application/AuthenticationService.java

```diff
@@ -29,6 +32,13 @@ public AuthenticationService(UserRepository userRepository,
         this.passwordEncoder = passwordEncoder;
     }
 
+    /**
+     * email과 password를 통해 인증하고, jwt 토큰을 리턴합니다.
+     *
+     * @param email 사용자의 이메일
+     * @param password 사용자의 비밀번호
+     * @return jwt token
```

Comment:
예외를 던지는 경우에는 JavaDoc에 꼭 `@throws`를 추가해 주세요.
```suggestion
     * @return jwt token
     * @throws LoginFailException 로그인에 실패한 경우
```